cmd/appliance/upgrade/prepare.go

```diff
@@ -170,8 +182,8 @@ func NewPrepareUpgradeCmd(f *factory.Factory) *cobra.Command {
 
 func checkImageFilename(i string) error {
 	// Check if its a valid filename
-	if rg := regexp.MustCompile(`(.+)?\d\.\d\.\d(.+)?\.img\.zip`); !rg.MatchString(i) {
-		return errors.New("Invalid name on image file. The format is expected to be a .img.zip archive with a version number, such as 5.5.1")
+	if rg := regexp.MustCompile(`\.img\.zip`); !rg.MatchString(i) {
+		return errors.New("Invalid name on image file. The format is expected to be a .img.zip archive.")
```

Comment:
file naming isn't as strict anymore. The only criteria is that the file ends with `.img.zip`